lib/decompress.ps1

```diff
@@ -240,7 +240,7 @@ function Expand-ZipArchive {
         $OriDestinationPath = $DestinationPath
         $DestinationPath = "$DestinationPath\_tmp"
     }
-    Expand-Archive -Path $Path -DestinationPath $DestinationPath -Force
+    Microsoft.PowerShell.Archive\Expand-Archive -Path $Path -DestinationPath $DestinationPath -Force
```

Comment:
Move this to a separate PR.

Reply:
Really don't know why there's people using Pscx v3.

Reply:
Because it's still in beta on PowerShell Gallery. Most people won't install beta software by default you know.

And you never know: https://github.com/ScoopInstaller/Scoop/pull/4638#discussion_r782685435 

`¯\_(ツ)_/¯`

Reply:
Yes since I don't use it. And, that's added by myself in #3425, sigh...